src/Microsoft.AspNet.Mvc.ViewFeatures/ViewFeatures/HtmlHelper.cs

```diff
@@ -882,6 +901,12 @@ protected virtual string GenerateDisplayText(ModelExplorer modelExplorer)
                 tagBuilder.WriteTo(ViewContext.Writer, _htmlEncoder);
             }
 
+            if (!suppressAntiforgery)
```

Comment:
Should `method == FormMethod.Get` also suppress the token? Could do that only in the extension methods with the `method` parameter but without `suppressAntiforgery`.